pw_scale.m

```diff
@@ -36,7 +41,23 @@
 % Revision history
 % 2016-03-19 - Fixed graph connectivity patch; Replaced UA weights with
 %              a conditional prior
+% 2017-09-13 - Refined the prior and code simplification
 
+if( ~exist( 'options', 'var' ) )
+    options = {};
+end
+
+opt = struct();
+
+% We don't the prior by default
+opt.prior = '';
```

Comment:
= 'none'

then in switch / case below - "otherwise" should generate an error
if the prior name is mistyped, the code should generate an error.